libscript/CMakeLists.txt

```diff
@@ -9,6 +9,17 @@ add_library(
     sources/Stack.cxx
 )
 
+target_compile_options(
+    ${PROJECT_NAME}
+    PRIVATE $<$<OR:$<CXX_COMPILER_ID:Clang>,$<CXX_COMPILER_ID:AppleClang>,$<CXX_COMPILER_ID:GNU>>:
+            -Wall -Wextra>
+            $<$<CXX_COMPILER_ID:MSVC>:
+            /W4
+            /wd4267
+            /wd4201
+            /wd4244>
+)
+
```

Comment:
Maybe put it in a function instead of duplicating it everywhere ?

Reply:
fixed in #189